### ISSUE TITLE
Consider supporting onMouseOver/Out

### ISSUE THREAD

**Issue by nkkollaw** [arve0/react-animate-on-change#30]
I had a use case (nav) where I eventually wrapped `<LI>`s with `<DIV>`s just to get onMouseOver/Out.

Would it make sense for react-animate-on-change to pass these events (perhaps onClick as well?)

It would have been great if I could have done:

```jsx
        <AnimateOnChange
          customTag="li"
          baseClassName="nav-item"
          animationClassName="animated bounce"
          animate={this.state.hover}
          onMouseOver={this.handleMouseOver} 
          onMouseOut={this.handleMouseOut}
        >
          <Link className={'nav-link' + (Boolean(this.props.active) ? ' active' : '')} to={this.props.to}>{this.props.children}</Link>
        </AnimateOnChange>
```

instead of:

```jsx
      <div style={{display: 'inline'}} onMouseOver={this.handleMouseOver} onMouseOut={this.handleMouseOut}>
        <AnimateOnChange
          customTag="li"
          baseClassName="nav-item"
          animationClassName="animated bounce"
          animate={this.state.hover}
        >
          <Link className={'nav-link' + (Boolean(this.props.active) ? ' active' : '')} to={this.props.to}>{this.props.children}</Link>
        </AnimateOnChange>
      </div>
```

**Comment by ModPhoenix** [comment]
Hi, I created a pool request #31 that would be able to forward custom props

**Comment by arve0** [comment]
Fixed in #31, released in `v2.2.0`.
